_extras/checkout.md

```diff
@@ -6,13 +6,30 @@ calendar: https://calendar.google.com/calendar/embed?src=oseuuoht0tvjbokgg3noh8c
 demopad: http://pad.software-carpentry.org/teaching-demos
 discussionpad: http://pad.software-carpentry.org/instructor-discussion
 ---
+
 After you have finished the two-day training class,
-you must go through these steps in order to complete your training and be certified:
+you must go through these steps in order to complete your training and be 
+fully certified as a Data and/or Software Carpentry instructor:
 
 1.  Submit a [small change to one of our lessons](#lesson-change).
 2.  Take part in an online [discussion session](#discussion-session).
 3.  Teach a short [demonstration lesson](#demo-lesson) online.
 
+These checkout steps aim to introduce you to key components of being a Software 
+and Data Carpentry instructor that are difficult to fully cover in the classroom 
+setting of the two-day training.  These are: 
+
+1.  Contributing to our lesson materials.  As our materials are developed by the 
+community, we want to ensure that all instructors know where and how to contribute 
+to our materials, particularly so that you can contribute in the future.  
```

Comment:
Add something about this also helping people become familiar with the lessons and how they work?